spec/services/creneaux_search/for_agent_spec.rb

```diff
@@ -103,29 +103,29 @@
       instance_double(
         AgentCreneauxSearchForm,
         organisation: organisation,
-        motif: motif,
-        service: motif.service,
+        motif: motif_by_phone,
+        service: motif_by_phone.service,
         agent_ids: [],
         team_ids: [],
         lieu_ids: nil,
         date_range: Date.new(2022, 10, 20)..Date.new(2022, 10, 30)
       )
     end
-    let(:motif) { create :motif, :by_phone, organisation: organisation }
-    let!(:plage_ouverture) { create(:plage_ouverture, motifs: [motif], first_day: Date.new(2022, 10, 25), lieu: nil, organisation: organisation) }
+    let(:motif_by_phone) { create :motif, :by_phone, organisation: organisation }
+    let!(:plage_ouverture) { create(:plage_ouverture, motifs: [motif_by_phone], first_day: Date.new(2022, 10, 25), lieu: nil, organisation: organisation) }
 
     it "has results" do
       expect(described_class.new(form).build_result.creneaux).to be_any
     end
 
-    describe "when there is concurrent PO with lieu" do
+    describe "when there is also another PO with lieu" do
       let(:lieu) { create(:lieu, organisation: organisation) }
       let(:motif_with_lieu) { create :motif, organisation: organisation }
-      let!(:plage_ouverture2) { create(:plage_ouverture, motifs: [motif, motif_with_lieu], first_day: Date.new(2022, 10, 20), lieu: lieu, organisation: organisation) }
+      let!(:another_po_with_lieu) { create(:plage_ouverture, motifs: [motif_by_phone, motif_with_lieu], first_day: Date.new(2022, 10, 20), lieu: lieu, organisation: organisation) }
 
-      it "give the good results with no conflict" do
-        # on utilise volontairement pas .first ici car les créneaux retournés ne sont pas triés
-        expect(described_class.new(form).build_result.creneaux.min_by(&:starts_at).starts_at).to eq(Time.zone.local(2022, 10, 20, 8, 0, 0))
+      it "give both creneaux" do
```

Comment:
👍 ce nom de cas est plus clair qu’avant

suggestion pour clarifier encore : `it "retourne des créneaux des deux plages d’ouvertures"`

et éventuellement dans ce cas rajouter un deuxième expect qui s’attend justement à trouver un créneau de l’autre plage d’ouverture ? surtout maintenant qu’on fait un `expect` `include` plutôt qu’un `min_by` ça me semblerait plus lisible

Reply:
Les 2 expects y sont déjà ;)